sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -412,6 +544,27 @@ ios_metrics AS (
   FROM
     metrics_org_mozilla_ios_fennec
 ),
+android_focus_metrics AS (
+  SELECT
+    *
+  FROM
+    metrics_org_mozilla_focus
+  UNION ALL
+  SELECT
+    *
+  FROM
+    metrics_org_mozilla_focus_beta
+  UNION ALL
+  SELECT
+    *
+  FROM
+    metrics_org_mozilla_focus_nightly
+  UNION ALL
+  SELECT
+    *
+  FROM
+    metrics_org_mozilla_klar
```

Comment:
Since we're here, should we also add Focus iOS and Klar iOS?

Reply:
@ANich are you planning on adding the iOS Focus/Klar as part of this PR, too? If not, would you mind to file a DO ticket about this?